recipe/meta.yaml

```diff
@@ -25,7 +25,7 @@ requirements:
 outputs:
   - name: cachecontrol
     build:
-      skip: true  # [py<36]
+      skip: true  # [py<37]
       script: {{ PYTHON }} -m pip install . -vv --no-deps
```

Comment:
```suggestion
      script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

Reply:
I tried this and got a build failure. Guessing it was due to the multi output?

Reply:
You need to also use flit_core in outputs.

Reply:
Done